baiji/config.py

```diff
@@ -76,7 +84,38 @@ def key(self):
     def secret(self):
         return self._try(['AWS_SECRET_ACCESS_KEY', 'AWS_SECRET'], 'AWS_SECRET')
 
+class Settings(Credentials):
+    '''
```

Comment:
This works. It's a little odd for settings to subclass credentials (i.e. doesn't satisfy an `is a` relationship), but the impact here is minor enough that I'm not too worried about it.